isis/src/base/objs/SpiceRotation/SpiceRotation.truth

```diff
@@ -656,7 +657,7 @@ CJ(3) = -8.6595606e-17 2.220446e-16 -1
 
 
 Testing loading cache from ALE ISD with time dependent quaternions and AV ...
-Has AV? Yes
+Has AV? No
```

Comment:
Why did this change?

Reply:
No sure, I thought it was changing from No to Yes. I'll take another look